app/src/main/java/com/eis/phoneringer/structure/ReceivedMessageListener.java

```diff
@@ -0,0 +1,56 @@
+package com.eis.phoneringer.structure;
+
+import android.content.Context;
+import android.util.Log;
+import android.widget.Toast;
+
+import androidx.annotation.NonNull;
+
+import com.eis.phoneringer.exceptions.IllegalPasswordException;
+import com.eis.smslibrary.SMSMessage;
+import com.eis.smslibrary.listeners.SMSReceivedListener;
+
+/**
+ * Listener used to manages incoming sms and evaluates if they are valid commands for the application
```

Comment:
I think this should be "Listener used to **manage** incoming **SMS** and **evaluate** if..."